templates/terraform/examples/base_configs/iam_test_file.go.erb

```diff
@@ -108,6 +108,9 @@ func TestAcc<%= resource_name -%>IamPolicyGenerated(t *testing.T) {
 	t.Parallel()
 
 <%= lines(compile('templates/terraform/iam/iam_context.go.erb')) -%>
+<% unless object.iam_policy.admin_iam_role.nil? -%>
+	context["service_account"] = getTestServiceAccountFromEnv(t)
```

Comment:
Can we put this inside `iam_context.go.erb`?

Reply:
`getTestServiceAccountFromEnv` will skip the test if the service account env var isn't set, so we should only do that on tests that are going to use it (in this case just the _policy tests).